addons/website/tests/test_attachment.py

```diff
@@ -41,3 +42,14 @@ def test_01_type_url_301_image(self):
 
     def test_02_image_quality(self):
         self.start_tour(self.env['website'].get_client_action_url('/'), 'website_image_quality', login="admin")
+
+    @mute_logger('odoo.addons.http_routing.models.ir_http', 'odoo.http')
```

Comment:
Why do you need to mute the logger here?

Reply:
> Why do you need to mute the logger here?

@detrouxdev I was inspired by https://github.com/odoo/odoo/commit/d7707a4a88b011686b5531edf54b08dc1a4952c7#diff-ab4a32b8da6461c750e479d0ddc235dc5c388110f80beae5f9d27cadcf34899c

Do you think it's unnecessary?

Reply:
I think if we don't know the actual reason, muting the logger might hide a sneaky error and we should probably avoid it, but if we have a valid reason it's okay. It's unfortunate that there's no comment on why the logger is muted for that other test.

Reply:
@bso-odoo Since you're the author of the test I was inspired by. Could you explain why you need to mute the logger in this case? (See the link above). Thanks.

Reply:
I'd say it was a mistake - and most likely copied from another test v_v

Reply:
I'll get rid of it ...